<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>register2</class>
 <widget class="QDialog" name="register2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_fillinfosellerregsiter">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fill in your information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nameseller">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ageseller">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>age:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_numseller">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_passseller">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_reenterpassseller">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Re-enter pass:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nameseller">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ageseller">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_numseller">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_passseller">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_reenterpassseller">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_submitregisterseller">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
